src/App.js

```diff
@@ -4,15 +4,57 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>
-          Eric F. Olsen
-        </h1>
-        <h3>
-          Software Engineer
-        </h3>
-      </header>
+      <Header />
+      {/* <Story />
```

Comment:
Rather than commenting this out.  Maybe create a new branch off of master to be a staging branch.  Then we can target staging with all the things aren't ready to go live.  Then when you're ready to go live with a set of changes, merge staging into master.

Reply:
Sounds good. I'll work on changing my workflow to use a staging step soon. For now, I broke each component into their own branch and removed all but Header from this PR.